src/components/molecules/user-result/user-result.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Avatar from 'components/atoms/avatar';
+
+import {
+  UserResultContainer,
+  AvatarContainer,
+  DetailsContainer,
+  StatsContainer,
+  Row,
+  NameDisplay,
+  UsernameDisplay,
+} from './user-result.styles';
+
+const UserResult = ({
+  avatarUrl,
+  email,
+  followersCount,
+  followingCount,
+  location,
+  login,
+  name,
+  starredRepositoriesCount,
+  url,
+}) => (
+  <UserResultContainer href={url} target="_blank">
+    <AvatarContainer>
+      <Avatar atl={login} src={avatarUrl} />
+    </AvatarContainer>
+    <DetailsContainer>
+      <Row>
+        <NameDisplay>{name}</NameDisplay>
+        <UsernameDisplay>{login}</UsernameDisplay>
+      </Row>
+      <Row>
+        <StatsContainer>Followers: {followersCount}</StatsContainer>
+        <StatsContainer>Following: {followingCount}</StatsContainer>
+        <StatsContainer>Starred: {starredRepositoriesCount}</StatsContainer>
+      </Row>
```

Comment:
The 2nd row was the only different piece between the 2 components so there's definitely a way to customize this piece but make a shared component as the base for the render. Only thing that could skew this would be the 1st and 3rd row rendering and if they should be different to make it obvious when a row represents a user v an org. Definitely would want to talk with a designer or stakeholder about this to figure out best course of action.